src/main/java/com/google/api/generator/gapic/composer/MethodSampleCodeHelperComposer.java

```diff
@@ -0,0 +1,167 @@
+    bodyExpr.add(responseExpr);
+
+    return TryCatchStatement.builder()
+        .setTryResourceExpr(assignClientVariableWithCreateMethodExpr(clientVarExpr))
+        .setTryBody(
+            bodyExpr.stream().map(e -> ExprStatement.withExpr(e)).collect(Collectors.toList()))
+        .setIsSampleCode(true)
+        .build();
+  }
+
+  // ==================================Helpers===================================================//
+
+  // Assign client variable expr with create client.
+  // e.g EchoClient echoClient = EchoClient.create()
+  private static AssignmentExpr assignClientVariableWithCreateMethodExpr(
+      VariableExpr clientVarExpr) {
+    return AssignmentExpr.builder()
+        .setVariableExpr(clientVarExpr.toBuilder().setIsDecl(true).build())
+        .setValueExpr(
+            MethodInvocationExpr.builder()
+                .setStaticReferenceType(clientVarExpr.variable().type())
+                .setReturnType(clientVarExpr.variable().type())
+                .setMethodName("create")
+                .build())
+        .build();
+  }
+
+  // Create a Map where key is method's argument name, and value is its VariableExpr.
+  private static Map<String, VariableExpr> createMethodArgumentsVariableExprs(
+      List<MethodArgument> arguments) {
+    return arguments.stream()
+        .collect(
+            Collectors.toMap(
+                methodArg -> methodArg.name(),
+                methodArg ->
+                    createVariableExpr(
+                        JavaStyle.toLowerCamelCase(methodArg.name()), methodArg.type())));
+  }
+
+  // Return a list of AssignmentExpr for method argument with its default value.
+  private static List<Expr> assignMethodArgumentsWithDefaultValues(
+      List<MethodArgument> arguments,
+      Map<String, VariableExpr> argVarExprs,
+      Map<String, ResourceName> resourceNames) {
+    return arguments.stream()
+        .map(
+            arg ->
+                createAssignmentExpr(
+                    argVarExprs.get(arg.name()),
+                    DefaultValueComposer.createDefaultValue(arg, resourceNames)))
+        .collect(Collectors.toList());
+  }
+
+  private static Expr createAssignExprForVariableWithClientMethod(
+      VariableExpr variableExpr,
+      VariableExpr clientVarExpr,
+      String methodName,
+      List<Expr> argumentsVarExprs) {
+    MethodInvocationExpr clientMethodInvocationExpr =
+        MethodInvocationExpr.builder()
+            .setExprReferenceExpr(clientVarExpr)
+            .setMethodName(JavaStyle.toLowerCamelCase(methodName))
+            .setArguments(argumentsVarExprs)
+            .setReturnType(variableExpr.variable().type())
+            .build();
+    return AssignmentExpr.builder()
+        .setVariableExpr(variableExpr.toBuilder().setIsDecl(true).build())
+        .setValueExpr(clientMethodInvocationExpr)
+        .build();
+  }
+
+  private static String getClientName(TypeNode clientType) {
```

Comment:
This wouldn't be the client name, since any type could be passed in, and it's simply converting that to lowercase. Could we inline this logic instead of adding a helper method, since it's straightforward?

Reply:
Similar to `createVariableExpr`
Flatten the code
```
    VariableExpr clientVarExpr =
        VariableExpr.withVariable(
            Variable.builder()
                .setName(JavaStyle.toLowerCamelCase(clientType.reference().name()))
                .setType(clientType)
                .build());
```
vs.
current code
```
VariableExpr clientVarExpr = createVariableExpr(getClientName(clientType), clientType);
```
Why not just use wrapper method where indicate the function to make the code easy to read?

Reply:
SG